news/2023/2023-12-12-how-advances-in-technology-affected-the-osu-community.md

```diff
@@ -0,0 +1,103 @@
+
+Mechanical keyboards have been a thing for decades, however after Razer put [gaming and mechanical switches into a compelling package](https://press.razer.com/product-news/the-worlds-first-mechanical-keyboard-designed-for-gaming-launched-at-gamescom-2010/), that launched the chain reaction that resulted in the mechanical keyboard market we know of today.
+
+By 2010 osu! players were already shifting away from clicking circles with their mouse, to clicking with their Z and X keys, so it would make sense that as these new mechanical keyboards were getting easier and cheaper to adopt into their setups, players would start using them for osu! as well. And while there's nothing stopping anyone from playing osu! with the membrane keyboards, the community at large is suggesting to newer players to purchase a mechanical keyboard, as it provides better feedback when a key is pressed.
+
+The mass adoption of mechanical keyboards did not only occur in the osu! community, but in the gaming community as a whole, and you'd be hard-pressed to find a gaming setup nowadays without one of these smack-dab in the middle of any gamer's desk.
+
+## High refresh rate LCD monitors (2012)
+
+Around mid 2012 first LCD high refresh rate monitors were released with the rates as high as 144hz. Even before LCD monitors were a thing CRT's had the capability of refreshing up to 100hz in most cases (and even higher for high-end models), however with the introduction of high refresh rate LCD panels this technology slowly became much easier to adopt, and is now a staple of any modern gaming setup.
+
+As osu! is a game with high amount of moving elements no matter the game mode (as shown below), high refresh rate monitors quickly became the next most-sought-after commodity piece of gear. While being an upgrade in comparison to 60hz monitors, and some people swearing by high refresh rate monitors helping them read higher approach rates & scroll speeds, it did not stop players who could not afford them from rising through the ranks and some of them even becoming top players such as [FGSky](https://osu.ppy.sh/users/2094566), regardless of the refresh rates of their monitor.
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="soonTM" frameborder="0" allowfullscreen></iframe>
+</div>
+
+## Touch screen's rise and fall (2015-2017)
+
+The earliest mentions of using touch screens as an input device came as soon as the game was released, again due to osu!'s roots as an *Osu! Tatakae! Ouendan* clone. Back then touch screens on PCs were not as prevalent, and most touch screen devices were "Tablet PCs", which were laptops that had a keyboard that can be rotated, folded, or outright detached, along with a touchscreen.
+
+It would take many years for the community to see the true potential of using touch screens in osu!, but starting in 2015 we started seeing glimpses, and eventually the peak of the play style in 2017 with the two players noted below.
```

Comment:
```suggestion
It would take many years for the community to see the true potential of using touch screens in osu!, but starting in 2015 we began seeing glimpses of this with Kuvster, and eventually the peak of the play style in 2017 with freedomdiver.
```

Reply:
```suggestion
It would take many years for the community to see the true potential of using touch screens in osu!, but starting in 2015 we began seeing glimpses of this with [Kuvster](https://osu.ppy.sh/users/4896624), and eventually the peak of the play style in 2017 with [Freedomdiver](https://osu.ppy.sh/users/8482062).
```
Since it's their first mention I'm gonna have to move where their profiles are linked